CSS/Menu/Menu.css

```diff
@@ -0,0 +1,45 @@
+@keyframes slidein {
+  0% {
+    left: -350px;
+  }
+  100% {
+    left: 0;
+  }
+}
+@keyframes slideout {
+  0% {
+    left: 0;
+  }
+  100% {
+    left: -350px;
+  }
+}
+.menu-button {
+  position: absolute;
+  left: 20px;
+  cursor: pointer;
+}
+.menu {
+  display: block;
+  position: fixed;
+  top: 75px;
+  left: -350px;
+  width: 350px;
+  height: 100vh;
+  /* This give the height 100% of the screen or, '100% of View Height' */
+  background-color: #81c784;
+  border: 1px solid #004d40;
+  border-left: none;
+  border-bottom: none;
+  z-index: 2;
+  /* This set the layer level of the element, it has precendence over initial level of 1*/
+}
+.menu ul li {
+  list-style-type: none;
+  font-size: 32px;
+  padding: 30px 0;
+  text-align: center;
+}
+.menu--open {
+  left: 0;
+}
```

Comment:
btw, it's always best to allow CSS to do its thing instead of trying to animate with JS.  It's just better built for it and JS animations just always look a little off.